executor/src/lib.rs

```diff
@@ -94,12 +94,13 @@ mod tests {
 
 	/// Default transfer fee
 	fn transfer_fee<E: Encode>(extrinsic: &E, fee_multiplier: Fixed64) -> Balance {
-		let length_fee = TransactionBaseFee::get() + TransactionByteFee::get() * (extrinsic.encode().len() as Balance);
+		let length_fee = TransactionByteFee::get() * (extrinsic.encode().len() as Balance);
```

Comment:
How this could have happened in the merge?

Reply:
The change is from `plug-blockchain/1.0.0-rc2`. I think the formula is correct now

Reply:
My question was rather about how we missed bringing this in, the first attempt? Is this an alert that the merge approach was not sound and so there could be more such issues or it was rather just a mistake?

Reply:
It was my mistake to not bringing in the change. I did this initial migration (https://github.com/cennznet/cennznet/pull/111) as quickly as I could so that other people can start working on it. And it makes sense that the change was picked up here